app/models/item_order.rb

```diff
@@ -7,4 +7,13 @@ class ItemOrder <ApplicationRecord
   def subtotal
     price * quantity
   end
+
+  def self.top_five
+      joins(:item).select('items.id, (items.name) as name, sum(item_orders.quantity) as quantity').group('items.id').order('quantity DESC').limit(5)
+  end
+
+  def self.bottom_five
+    joins(:item).select('items.id, (items.name) as name, sum(item_orders.quantity) as quantity').group('items.id').order('quantity ASC').limit(5)
+  end
```

Comment:
`top_five` and `bottom_five` pretty much the exact same except for the order at the end.

Reply:
These look great!

Reply:
Great Job